plugins/qeta-backend/migrations/20230425-attachments.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function up(knex) {
+  await knex.schema.createTable('question_attachments', table => {
```

Comment:
I would change this to just 'attachments' as the 'question_attachments' means a join table in my opinion. Also users probably should be able to upload images also to answers, right?